### PR TITLE
encourage the creation of custom profile components

### PR DESCRIPTION
**Pull request by charleslavon**
This PR updates ProfilePage to redirect to an accountId's custom profile page if it exists.

### CODE REVIEW

File: src/ProfilePage.jsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 
+// if it exists, render the accountId's custom profile component
+if(Social.getr(`${accountId}/widget/ProfilePage`)){
+  return (<Widget 
+            src={`${accountId}/widget/ProfilePage`}
+            props={{
+              accountId
+            }}
+          />)
+}
```
Comment (shelegdmitriy) [inline src/ProfilePage.jsx:14]: @charleslavon could you please apply formatting here?

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

**Review by gabehamilton — APPROVED** [review]

**Comment by gabehamilton** [comment]
Was thinking about this more. Maybe this could use a wrapper (tabs?) such that the viewing user can switch between the custom profile component and the standard one? The standard one may serve as a source of truth about users.

**Review by shelegdmitriy — APPROVED** [review]

**Comment by shelegdmitriy** [comment]
@gabehamilton sounds interesting. But let's move this to the discussions, @charleslavon WDYT?

**Comment by thisisjoshford** [comment]
uh-mazing @charleslavon ! 💪 Thank you so much for putting some time in on your weekend off to hack on this! 🙏

@gabehamilton I like your idea for adding more options here (re: tabs/toggle) and also have some ideas as far as wanting to standardize _some_ things on the profile page such as contact info, contribution graph, profile image, sponsor/tip button etc. This way the most common things people want to find are highly discoverable and people don't have to play "Where's Waldo" if they wanna reach out to a dev or tip them.

I'm a big fan of being Aglie and shipping small incremental improvements rather than blocking progress here. IMO we should ship this as is (or make a minor tweak if needed) and then think more deeply about this with the design team (@heycorwin @sarahmills) 

cc @ravigautham

**Review by thisisjoshford — APPROVED** [review]
uh-mazing @charleslavon ! 💪 Thank you so much for putting some time in on your weekend off to hack on this! 🙏

@gabehamilton I like your idea for adding more options here (re: tabs/toggle) and also have some ideas as far as wanting to standardize _some_ things on the profile page such as contact info, contribution graph, profile image, sponsor/tip button etc. This way the most common things people want to find are highly discoverable and people don't have to play "Where's Waldo" if they wanna reach out to a dev or tip them.

I'm a big fan of being Aglie and shipping small incremental improvements rather than blocking progress here. IMO we should ship this as is (or make a minor tweak if needed) and then think more deeply about this with the design team (@heycorwin @sarahmills) 

cc @ravigautham

**Pull request merged**